charts/gardener/gardenlet/charts/runtime/templates/configmap-componentconfig.yaml

```diff
@@ -123,10 +123,11 @@ data:
       shootStateSync:
         concurrentSyncs: {{ required ".Values.global.gardenlet.config.controllers.shootStateSync.concurrentSyncs is required" .Values.global.gardenlet.config.controllers.shootStateSync.concurrentSyncs }}
         syncPeriod: {{ required ".Values.global.gardenlet.config.controllers.shootStateSync.syncPeriod is required" .Values.global.gardenlet.config.controllers.shootStateSync.syncPeriod }}
-      {{- if .Values.global.gardenlet.config.controllers.shootedSeedRegistration }}
-      shootedSeedRegistration:
-        {{- if .Values.global.gardenlet.config.controllers.shootedSeedRegistration.syncJitterPeriod }}
-        syncJitterPeriod: {{ .Values.global.gardenlet.config.controllers.shootedSeedRegistration.syncJitterPeriod }}
+      {{- if .Values.global.gardenlet.config.controllers.managedSeed }}
```

Comment:
Yes. Is a somewhat more detailed release note in the PR enough or do you have something else in mind? Is there an example of a "good" similar release note?

Reply:
Hm, I think something like this would be good enough:

\`\`\`action operator
The `.controllers.shootedSeedRegistration` field has been removed from the `GardenletConfiguration` in favor of the newly introduced `ManagedSeed` controller (configurable via `.controllers.managedSeed`). Please adapt your Gardenlet Helm chart values and/or example Gardenlet configuration files.
\`\`\`

Reply:
OK, thanks, added.